### PR TITLE
BUG: stats: Inconsistency in the multivariate_normal docstring #6263

### LINKED ISSUE #6263: BUG: stats: Inconsistency in the multivariate_normal docstring
The docstring for `scipy.stats.multivariate_normal` (actually in `multivariate_normal_gen`) includes this:

    Methods
    -------
    ``pdf(x, mean=None, cov=1, allow_singular=False)``
        Probability density function.
    ``logpdf(x, mean=None, cov=1, allow_singular=False)``
        Log of the probability density function.
    ``rvs(mean=None, cov=1, size=1, random_state=None)``
        Draw random samples from a multivariate normal distribution.

But the `pdf` and `logpdf` methods do not have default values for `mean` and `cov`.  E.g.

    def pdf(self, x, mean, cov, allow_singular=False):
        ...

See http://stackoverflow.com/questions/37798025/multivariate-normal-pdf-in-scipy for a report of this problem out in the wild.

### PR DESCRIPTION
**Pull request by bhavika**
Attempting to close [#6263](https://github.com/scipy/scipy/issues/6263).

I've added a test that covers `logpdf` tests using the `mean` and `cov` values specified in the docstring for `multivariate_normal_gen`.

### CODE REVIEW

File: scipy/stats/tests/test_multivariate.py [hunk 1]
```diff
@@ -64,6 +64,15 @@ def test_logpdf(self):
...
         assert_allclose(d1, np.log(d2))
 
+    def test_logpdf_default_values(self):
+        # Check that the log of the pdf is in fact the logpdf
+        # with default parameters Mean=None and cov = 1
+        np.random.seed(1234)
+        x = np.random.randn(5)
+        d1 = multivariate_normal.logpdf(x)
+        d2 = multivariate_normal.pdf(x)
+        assert_allclose(d1, np.log(d2))
```
Comment (rgommers) [inline scipy/stats/tests/test_multivariate.py]: Here you test only that ``pdf`` and ``logpdf`` have consistent defaults. Probably you also want to test that the defaults are what they should be. So test ``(x)`` against ``(x, None, 1)``

**Comment by rgommers** [comment]
The extra test would be nice to have, other than that this looks good to merge. Thanks @bhavikat

**Comment by bhavikat** [comment]
I can add the extra test but I'm not sure what would be a good name for it. Since we're still testing for default values, I can't think of anything better than 'test_logpdf_default_values_2' but I'm not sure this is acceptable. Is it okay if I combine the two tests?

**Comment by rgommers** [comment]
Combining the two tests is perfectly fine.

**Comment by bhavikat** [comment]
Done. Let me know if this is correct. Thank you @rgommers.

**Comment by rgommers** [comment]
Merged, thanks @bhavikat. Keep them coming!

**Pull request merged**